src/libs/actions/Task.js

```diff
@@ -119,6 +119,24 @@ function createTaskAndNavigate(parentReportID, title, description, assignee, ass
                 isOptimisticReport: false,
             },
         },
+        {
```

Comment:
Can you inverse the order of those two onyx instructions? Update `optimisticTaskCreatedAction` then `optimisticAddCommentReport` (just for readability and to be consist with how other onyx are structured). Also make the value in one line as it's in `failureData`.